Ladybird/AppKit/UI/LadybirdWebView.mm

```diff
@@ -623,7 +623,8 @@ - (void)setWebViewCallbacks
                                item:item];
         }
 
-        auto* event = Ladybird::create_context_menu_mouse_event(self, content_position);
+        auto device_pixel_ratio = m_web_view_bridge->device_pixel_ratio();
```

Comment:
Let's improve the commit title/message to explain why this was wrong. "Fix ... bug" leaves something to be desired.

From CONTRIBUTING.md:
```
Don't:
* Use weasel-words like "refactor" or "fix" to avoid explaining what's being changed.
```

Do all context menus have this issue?

Reply:
No only the select dropdown was effected